webclient/src/views/view/view-view.inc

```diff
@@ -8,6 +8,32 @@
          class="img-responsive"/>
   </a>
 
+  <!-- Pending coordinates counter (if any) -->
+  <div class="panel coord-counter" v-if="coord_counter.counter">
+    <div class="panel-body columns">
+      <div class="column col col-sm-12 msg">
+        ${{_.view_view.msg.coordinate-counter.msg-1}}$
+        <em>{{ coord_counter.counter }}</em>
+        ${{_.view_view.msg.coordinate-counter.msg-2}}$
+        <button class="btn btn-action btn-sm btn-link tooltip tooltip-left"
```

Comment:
On the right ther is more free space ;)
```suggestion
        <button class="btn btn-action btn-sm btn-link tooltip tooltip-right"
```

Reply:
I think we have to test this in practice, because on mobile (for me) the info button is on the right, so I need to open it to the left.